SpatialConnect/GeopackageStore.m

```diff
@@ -249,6 +284,43 @@ - (SCPolygon *)coverage:(NSString *)layer {
   return [ts coveragePolygon];
 }
 
+- (RACSignal *)send:(SCSpatialFeature *)feature {
+  SpatialConnect *sc = [SpatialConnect sharedInstance];
+  if (sc.backendService.status == SC_SERVICE_RUNNING) {
+    SCMessage *msg = [[SCMessage alloc] init];
+    msg.payload = [[feature JSONDict] JSONString];
+    [[[sc.backendService publishReplyTo:msg onTopic:[self syncChannel]] filter:^BOOL(SCMessage *m) {
+      //TODO check response message to see if feature was successfully uploaded
+      return YES;
+    }] flattenMap:^RACSignal*(id x) {
+      return [self sendComplete:feature];
+    }];
+    //temporaily assume message has been sent
+    return [self sendComplete:feature];
+  } else {
+    return [RACSignal empty];
+  }
+}
+
+- (NSArray *)unSent {
```

Comment:
Probably make this a signal so that all syncing interfaces return an observable.